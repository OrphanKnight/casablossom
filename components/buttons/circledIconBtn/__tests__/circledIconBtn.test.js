import { render } from '@testing-library/react';

import CircledIconBtn from '../index';

it('renders CircledIconBtn component', () => {
  const props= {type:"submit", text:"Sign in"}
  const { container } = render(<CircledIconBtn {...props} />);
  expect(container).toMatchSnapshot();
})