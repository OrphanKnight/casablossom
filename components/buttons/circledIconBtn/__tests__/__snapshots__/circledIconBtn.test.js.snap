// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CircledIconBtn component 1`] = `
<div>
  <button
    class="button"
    type="submit"
  >
    Sign in
    <div
      class="svg__wrap"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 24 24"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="m11.293 17.293 1.414 1.414L19.414 12l-6.707-6.707-1.414 1.414L15.586 11H6v2h9.586z"
        />
      </svg>
    </div>
  </button>
</div>
`;
